src/styles/blocks/button.css

```
@@ -0,0 +1,36 @@
+.button {
+  align-self: center;
+  border: 1px solid #000;
+  background: #fcffe7;
+  padding: 9px 120px;
+}
+
+.button--white {
+  background: white;
+}
+
+.button--big {
+  padding: 9px 140px;
+}
```

```suggestion
.button--big {
    max-width: 300px;
    padding: 9px 10px;
    width: 100%;
}
```

можно так размер кнопкам еще менять, но это как пример